pkg/util/discord.go

```diff
@@ -79,3 +80,12 @@ func GenerateUpvoteMessage(discordID, source string) *upvoteMsg {
 	ran := rand.Intn(len(presets) - 1)
 	return &presets[ran]
 }
+
+func RetryRequest(handler func() error) error {
```

Comment:
we should allow to pass `interval` to this func
```suggestion
func RetryRequest(handler func() error, interval time.Time) error {
```

Reply:
cool, I will add it